src/gui_elements.py

```diff
@@ -0,0 +1,217 @@
+import os
+import pygame
+from typing import Callable
+from pygame.locals import Rect
+from gui_utility import center_relative_to
+from cell import Clickable
+
+
+class StylizedText:
+    def __init__(self, position: pygame.Rect, content: str = '',
+                 text_colour: pygame.color.Color = pygame.color.Color(255, 255, 255),
+                 font_family: str = 'arial', font_size: int = 24,
+                 font_style: int = 0) -> None:
+        """
+        :param content: Содержимое.
+        :param position: Позиция.
+        :param text_colour: Цвет.
+        :param font_family: Шрифт текста.
+        :param font_size: Размер текста.
+        :param font_style: Стиль текста. Задаётся битовой маской: 0b01 - жирный, 0b10 - курсив, 0b100 - подчёркивание.
+        :return:
+        """
+        self._position: pygame.Rect = position
+        self._content: str = content
+        self.text_colour: pygame.color.Color = text_colour
+        self.font_family: str = font_family
+        self.font_size: int = font_size
+        self.font_style: int = font_style
+        self.__text: list[tuple[pygame.SurfaceType, pygame.Rect]] = self.__create_text()
+
+    @property
+    def content(self) -> str:
+        return self._content
+
+    @content.setter
+    def content(self, text: str) -> None:
+        if text is None:
+            raise TypeError('content must be str')
```

Comment:
Как is None связан с content must be str?
```suggestion
        if not isinstance(text, str):
            raise TypeError('Content must be a string type')
```